examples/graphql-caching-global/graphql_caching_global.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/graphql;
+
+// Defines a `record` type to use as an object in the GraphQL service.
+type User readonly & record {|
+    int id;
+    string name;
+    int age;
+|};
+
+// Defines an in-memory table to store the profiles.
+table<User> key(id) users = table [
+    {id: 1, name: "Walter White", age: 50},
+    {id: 2, name: "Jesse Pinkman", age: 25}
+];
+
+// The `cacheConfig` in the `graphql:ServiceConfig` annotation is used to 
+// configure the cache for the GraphQL service.
+// The `enabled` field enables/disables the cache for the field.
+// The `maxAge` field sets the maximum age of the cache in seconds. (default: 60)
+// The `maxSize` field indicates the maximum capacity of the cache table by entries. (default: 120)
+@graphql:ServiceConfig {
+    cacheConfig: {}
```

Comment:
Shall we use different values here since we have already used default values in field caching BBE?